[jira-ticket]/[build-id] (#247)
spotinst/terraform-provider-spotinst: website/docs/r/elastigroup_aws_suspend_processes.html.markdown

---
layout: "spotinst"

> page_title: "Spotinst: elastigroup_aws_suspend_processes"

Reviewer: ```suggestion
page_title: "Spotinst: elastigroup_aws_suspension"
```

subcategory: "Elastigroup"
description: |-

>   Provides a Processes suspensions to Spotinst AWS group resource.

Reviewer: ```suggestion
  Provides a process suspensions to Spotinst AWS group resources.
```

---

> # spotinst\_elastigroup\_aws\_susoend_processes

Reviewer: ```suggestion
# spotinst\_elastigroup\_aws\_suspension
```
